src/Workspaces/VisualBasic/Portable/Recommendations/VisualBasicRecommendationService.vb

```diff
@@ -71,6 +73,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Recommendations
             Return SpecializedCollections.EmptyEnumerable(Of ISymbol)()
         End Function
 
+        Private Function GetSymbolsForGlobalStatementContext(
+            context As VisualBasicSyntaxContext,
+            cancellationToken As CancellationToken
+        ) As IEnumerable(Of ISymbol)
```

Comment:
The entire recommendation service (both C# and VisualBasic) use `IEnumerable<ISymbol>` to return the list of completions.